<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TallyAccount</class>
 <widget class="QWidget" name="TallyAccount">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1289</width>
    <height>749</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>IPC</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="7">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="9">
    <widget class="QTableWidget" name="tableWidget"/>
   </item>
   <item row="0" column="8">
    <widget class="QPushButton" name="pushButton_outaccount">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>35</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>35</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/res/png/buttons/120_35.png);border:none;color: rgb(255,255,255);font-size : 16px;</string>
     </property>
     <property name="text">
      <string>out account</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Total income:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QLabel" name="label_punishment">
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>punishment</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_totalincome">
     <property name="text">
      <string>0.0</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_neterror">
     <property name="styleSheet">
      <string notr="true">font-size : 14px;
color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>net error</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="5">
    <widget class="QLabel" name="label_shownumtip">
     <property name="text">
      <string>Show the last 20 Accounting records. If you want to see more, you can go to the browser to view the block.</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
